[caffe2][be][2/n] migrate gloabl  static initializer

klein-meta opened the pull request:
Summary:
Caffe2 lib has 200+ global static initializer usage, which are papar-cut reference to startup perf. Detail in this post https://fb.workplace.com/groups/arglassesperf/permalink/623909116287154.
Kick off a stack to migirate all usage of global static initializer in caffe2.

Test Plan: CI

Differential Revision: D57758185

aaronenyeshi reviewed (changes requested):
please fix the failing unit tests

klein-meta committed:
[caffe2][be][2/n] migrate gloabl static initializer (#127620)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/127620

Caffe2 lib has 200+ global static initializer usage, which are papar-cut reference to startup perf. Detail in this post https://fb.workplace.com/groups/arglassesperf/permalink/623909116287154.
Kick off a stack to migirate all usage of global static initializer in caffe2.

Test Plan: CI

Differential Revision: D57758185
